gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+- **CouncilArbitrator:** Will be called by The Graph's Arbitrator with the result of a dispute to solve it so that it can be closed.
+
+## Off-Chain Agents
+
+Optimistic EBO allows any off-chain agent to interact with it by creating requests, proposing responses, or disputing responses in a permissionless manner.
+
+### Architecture
+
+The EBO Agent will be implemented as a single and standalone process that runs on a dedicated instance. The primary responsibility of this process is to poll events from the relevant on-chain smart contracts on the Protocol chain (Arbitrum) and respond to these events with specific actions tailored to the flow requirements. Additionally, the agent will interface with blockchains supported by The Graph and perform block computations for each epoch as necessary.
+ 
+
+> When ready, we will suggest that the Indexers run the instances. There is an intrinsic incentive for them to post EBO updates as they use them to collect indexing rewards. Future changes to issuance and incentives could include other ways to incentivize EBO operation.
+> 
+
+### System limitations
+
+The system will not handle requests that escalate to the Arbitrator for final resolution, as this isn’t automated by the off-chain agent and the Arbitrator must resolve the dispute manually. Additionally, once an epoch concludes, the system will not manage any pending requests.
+
+![EBO-high-level.jpeg](../assets/gip-0080/Optimistic%20EBO%20high-level.jpeg)
+
+The development of the EBO Agent will consist of two modules: 
+
+- **DisputeModule:** The DisputeModule will handle all dispute-related activities and will operate in a fully reactive manner based on incoming events. It is important to note that all dispute logic will occur on the protocol chain. Hence, the DisputeModule will interact exclusively with the protocol chain, which in this case is Arbitrum.
+- **BlockNumberModule:** The BlockNumberModule will be responsible solely for computing blocks based on timestamps on the blockchains supported by The Graph, including both EVM and non-EVM chains. For EVM chains, RPC calls will be utilized to extract the necessary data. For non-EVM chains, a third-party service.
+
+# **Dependencies**
+
+The [horizon staking contract](https://github.com/graphprotocol/contracts/pull/944), as described in GIP-0066, needs to be deployed to integrate with it.
+
+# **Validation**
+
+### **Testnet**
+
+We propose a trial period on testnet for the new Optimistic EBO implementation, with the participation of volunteer indexers running the new instances. The goal is to test as many connected chains to the system as possible.
+
+### **Internal Reviews**
```

Comment:
Great GIP, as you can tell my feedback is mainly minor tweaks rather than seeing anything of substance wrong.

I wonder if we want to avoid the 'process and planning' of this section being a part of a GIP, and try to keep the GIP about a technical proposal. Not a big deal in this case, but what if those plans or details change? Do we update the GIP? If so, unnecessary churn, if not, cannot rely on what is written here, and it might mislead.

Reply:
Thank you @RembrandtK. Agree that it's a good idea to remove this! We are also removing the Testnet phase as it's not mandatory and it's something we will discuss at a later stage